llvm/test/CodeGen/PowerPC/git_revision.ll

```diff
@@ -0,0 +1,13 @@
+; Check that the git revision is contained in the assembly/object files
+
+; RUN: cd %S && git rev-parse HEAD > %t
```

Comment:
I've tried `llc --version`, but it doesn't return the revision unfortunately. I'll try to find some other ways.

Reply:
Also, it's possible the user built LLVM is such as way as to disable the hash (i.e. with `LLVM_APPEND_VC_REV` off), in which case I'm assuming this test won't find it. You'd need something extra like a lit feature flag to control this.

(It might not be worth the trouble to add that just for this purpose)